bin/craft

```diff
@@ -184,6 +196,8 @@ function start {
     fi
 }
 
+check_nofile_limit
```

Comment:
Can't we move this call to the `start` function? This doesn't need to be evaluated on every call of the script but rather only once when we actually start a container